public/app/plugins/panel/graph/data_processor.ts

```diff
@@ -20,7 +20,7 @@ export class DataProcessor {
       return list;
     }
 
-    for (const series of dataList) {
+    for (const [index, series] of dataList.entries()) {
```

Comment:
minor nit -- that I know @torkelo agrees with...  what about just a regular `for(let i=0,...` loop?

Reply:
yea, commented on this one in the other PR (this is part of the other PR as well). entries() seems like a scary function to use for loops as it creates a whole new array with index,value pairs.